azure-pipelines-full-tests.yml

```diff
@@ -16,63 +16,6 @@ variables:
 - template: ${{ variables.Pipeline.Workspace }}/.azure-pipelines/templates/variables.yml
 
 jobs:
```

Comment:
What's the reason behind the deletion of these three profile tests?

Reply:
This three profiles are not required as the azure stack profiles are moved in the core

Reply:
Never mind, I found the explanation.